src_py/mixer.py

```
@@ -0,0 +1,829 @@
+            raise ValueError("distance out of range, expected (0, 255)")
+
+        x = math.sin(angle) * distance
+        z = math.cos(angle) * distance
+        self._track.set_3d_position((x, 0, z))
+
+    def set_volume(self, arg1: float, arg2: float | None = -1.0, /):
+        if arg2 == -1.0:
+            self._track.gain = arg1
+            return
+
+        if arg2 == None:
```

Flake8 (7.3.0)

[error] comparison to None should be 'if cond is None:'

(E711)